core/conn/jdbcT4/pom.xml

```diff
@@ -27,11 +27,35 @@
 -->
 
   <modelVersion>4.0.0</modelVersion>
+  <!--<parent>
+    <groupId>org.apache.trafodion</groupId>
+    <artifactId>trafodion</artifactId>
+    <version>1.3.0</version>
```

Comment:
@xwq i refered our parent pom:  
<groupId>org.apache.trafodion</groupId>
  <artifactId>trafodion</artifactId>
  <version>1.3.0</version>
  <packaging>pom</packaging>
it's not necessary, in case any negative effects, i comment them.